app/src/main/java/lt/getpet/getpet/adapters/OnboardingAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package lt.getpet.getpet.adapters
+
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import lt.getpet.getpet.constants.ActivityConstants.Companion.POSITION
+import lt.getpet.getpet.fragments.OnBoardingFragment
+
+
+/**
+ * Created by Greta Grigutė on 2018-11-10.
+ */
+class OnboardingAdapter(private val mContext: Context, fm: FragmentManager) : FragmentPagerAdapter(fm) {
+
+    private var mBundle: Bundle? = null
+    private var mFragment: Fragment? = null
+    private var mPosition: Int? = null
```

Comment:
Don't use m prefix and follow Google style guide https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names
```
In Google Style, special prefixes or suffixes are not used. For example, these names are not Google Style: name_, mName, s_name and kName.
```

Reply:
True, wrote like that thinking is it good to use naming like this or not :) now I have an answer, thank you.